### PR TITLE
SC-3909 - teacher cannot create users anymore.

### PR DESCRIPTION
**Pull request by mwolowyk-cap**
# Description

Teacher cannot create users any more. So it cannot create users to add them to the team. Users should be created before and then added to the team by the teacher.

## Approval for review
- [ ] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### Link to Definiton of Done
More and detailed information on the *definition of done* can be found [on Confluence](https://docs.schul-cloud.org/pages/viewpage.action?pageId=92831762)

### CODE REVIEW

**Comment by CordlessWool** [comment]
please link the ticket

File: features/createTeam.feature [hunk 1]
```diff
@@ -6,12 +6,12 @@ Background: a teacher logged in and created a new pupil
...
-
+	# TODO: teacher cannot create students any more. Please adjust this scenario
 Scenario Outline: create a team with two new members
-When teacher adds a new student with <firstname1>, <lastname1>, <email1>
-When teacher adds one more student with <firstname2>, <lastname2>, <email2>
+#When teacher adds a new student with <firstname1>, <lastname1>, <email1>
+#When teacher adds one more student with <firstname2>, <lastname2>, <email2>
 When teacher creates a new team with <teamname> and
-When teacher adds two students to this team
+#When teacher adds two students to this team
```
Comment (CordlessWool) [inline features/createTeam.feature:14]: should still be tested, maybe use the seed data for it

**Comment by yuliashikhareva** [comment]
Please be aware that you cannot merge it before it was merged in other repos - I have seen that PR in Client is still open...

**Comment by mwolowyk-cap** [comment]
Is not necessary any more due to https://github.com/hpi-schul-cloud/integration-tests/pull/141

**Comment by mwolowyk-cap** [comment]
Is not necessary any more due to #141

**Pull request closed without merging**